pkgs/top-level/all-packages.nix

```diff
@@ -9885,6 +9889,11 @@ in
     self = python35Packages;
   });
 
+  python36Packages = recurseIntoAttrs (callPackage ./python-packages.nix {
```

Comment:
also not `recurseIntoAttrs` here. I don't think we should make the packages visible yet.